ludwig/models/modules/loss_modules.py

```diff
@@ -478,6 +494,29 @@ def weighted_softmax_cross_entropy(
             reduction=Reduction.NONE)
     return train_loss
 
+def sigmoid_cross_entropy(
+        logits,
+        targets,
+        class_weights=1,
+        labels_smoothing=0,
+        **kwargs
+):
+    use_class_weights = not isinstance(class_weights, (int, float))
+    if use_class_weights:
+        train_loss = softmax_cross_entropy_with_class_weighting(
```

Comment:
this is softmax not signoid

Reply:
don't actually need this helper, let me clean up